arangod/Graph/Enumerators/TwoSidedEnumerator.h

```diff
@@ -100,6 +100,8 @@ class TwoSidedEnumerator {
 
     auto provider() -> ProviderType&;
 
+    auto getGraphOptions() -> GraphOptions;
```

Comment:
I think this holds true for all Enumerators in this PR:
What are the GraphOptions for?
Is that only for isDisjoint? I have comments to move that somewhere else.

Reply:
This got removed completely due to: Refactored the API. isDisjoint and isSatelliteLeader now part of PathValidatorOptions